src/Admin.php

```diff
@@ -51,107 +53,95 @@ private function __clone()
+                "Please contact your server administrator.";
         }
-        if (version_compare(phpversion(), '5.4.0', '<')) {
-            $errors[] = 'Yoti could not be installed. Yoti PHP SDK requires PHP 5.4 or higher.';
+        if (version_compare(phpversion(), '7.2', '<')) {
+            $errors[] = 'Yoti could not be installed. Yoti PHP SDK requires PHP 7.2 or higher.';
         }
 
         // Get data
-        $data = $config;
+        $data = $config->load();
         $updateMessage = '';
 
-        if ($_SERVER['REQUEST_METHOD'] == 'POST')
-        {
-            try
-            {
+        if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+            try {
                 $this->setPostData();
 
                 $data['yoti_app_id'] = trim($this->postVar('yoti_app_id'));
                 $data['yoti_scenario_id'] = trim($this->postVar('yoti_scenario_id'));
                 $data['yoti_sdk_id'] = trim($this->postVar('yoti_sdk_id'));
                 $data['yoti_company_name'] = trim($this->postVar('yoti_company_name'));
-                $data['yoti_delete_pem'] = $this->postVar('yoti_delete_pem') ? TRUE : FALSE;
+                $data['yoti_delete_pem'] = $this->postVar('yoti_delete_pem') ? true : false;
                 $data['yoti_only_existing'] = $this->postVar('yoti_only_existing');
                 $data['yoti_user_email'] = $this->postVar('yoti_user_email');
                 $data['yoti_age_verification'] = $this->postVar('yoti_age_verification');
-                $pemFile = $this->filesVar('yoti_pem', $config['yoti_pem']);
+                $pemFile = $this->filesVar('yoti_pem', $config->get('yoti_pem'));
 
                 // Validation
-                if (!$data['yoti_app_id'])
-                {
+                if (!$data['yoti_app_id']) {
                     $errors['yoti_app_id'] = 'App ID is required.';
                 }
-                if (!$data['yoti_sdk_id'])
-                {
+                if (!$data['yoti_sdk_id']) {
                     $errors['yoti_sdk_id'] = 'Client SDK ID is required.';
                 }
-                if (empty($pemFile['name']))
-                {
+                if (empty($pemFile['name'])) {
                     $errors['yoti_pem'] = 'PEM file is required.';
-                }
-                elseif (!empty($pemFile['tmp_name']) && !openssl_get_privatekey(file_get_contents($pemFile['tmp_name'])))
-                {
+                } elseif (
+                    !empty($pemFile['tmp_name']) &&
+                    !openssl_get_privatekey(file_get_contents($pemFile['tmp_name']))
+                ) {
                     $errors['yoti_pem'] = 'PEM file is invalid.';
                 }
-            }
-            catch (\Exception $e) {
+            } catch (\Exception $e) {
                 $errors['yoti_admin_options'] = 'There was a problem saving form data. Please try again.';
             }
 
             // No errors? proceed
```

Comment:
Not essential to change now, but I was never a fan of these comments which can easily be ascertained by the line of code below! 

Might be more important in PHP, but with stricter typing languages like .NET, where much more information is able to be gleaned from the code alone, we followed guidelines like this - https://blog.codinghorror.com/coding-without-comments/

Reply:
More than happy to remove comments like this!

Reply:
I'll open a separate PR tidying up comments/docs